movieDetails.js

```diff
@@ -0,0 +1,242 @@
+import { moviesDatabase } from "./movieStorage.js";
+
+// Create movie cards
+function createMovieCard(movie) {
```

Comment:
This function is quite long. You could split it into multiple parts:
1. Display the movie info
2. Create the star rating component
3. Create the comment component